server/views/account/email-signin.jade

```diff
@@ -34,18 +35,13 @@ block content
                   a(href="/deprecated-signin") Or click here if you want to sign in with other options.
 
         script.
-           $(document).ready(function() {
-
-             function disableMagicButton (isDisabled) {
-               if (isDisabled) {
-                 $('#magic-btn')
-                   .html('<span class="fa fa-circle-o-notch fa-spin fa-fw"></span>')
-                   .prop('disabled', true);
-               } else {
-                 $('#magic-btn')
-                   .html('<span class="fa.fa-envelope">Get a magic link to sign in.</span>')
-                   .prop('disabled', false);
-               }
+          $(document).ready(function() {
+            function disableMagicButton () {
+              var userEmail = $("input[name=email]").val();
+              $('#sign-in-form')
+               .empty()
+               .html('<h3 class="text-center"> OK - we are sending a magic link to ' + userEmail + ' </h3>')
```

Comment:
We can't say we are sending an email just because the submit action has occurred. This is misleading and false.
  
The original code seemed to add a spinner to the button but it doesn't seem to appear. If we could get this working and adding a message stating that an 'attempt' to send an email is being made. That should be sufficient.